### PR TITLE
JEN-910 Changes for 8.0 Valgrind job

### PR DESCRIPTION
**Pull request by abychko**
[*] changed MTR --parallel to cores/2 for valgrind builds
[*] increased build timeout to 15h
[+] added new platforms to the matrix

### CODE REVIEW

**Comment by delgod** [comment]
why we need two times less threads in all params jobs?

**Comment by abychko** [comment]
> why we need two times less threads in all params jobs?

only if [[ "${ANALYZER_OPTS}" == *WITH_VALGRIND=ON* ]]

File: jenkins/pipeline.groovy [hunk 1]
```diff
@@ -105,7 +105,7 @@ pipeline {
         compressBuildLog()
         skipDefaultCheckout()
         skipStagesAfterUnstable()
-        timeout(time: 10, unit: 'HOURS')
+        timeout(time: 15, unit: 'HOURS')
```
Comment (laurynas-biveinis) [inline jenkins/pipeline.groovy]: This is global right? Possible to limit to Valgrind jobs only?

Comment (abychko) [reply jenkins/pipeline.groovy]: yes, should be possible to use variable with default value

Comment (laurynas-biveinis) [reply jenkins/pipeline.groovy]: On the other hand, looking at sanitizer run results, please set 15 h there too

**Review by laurynas-biveinis — APPROVED** [review]
Looks good, let's take a look at a Valgrind test job: start and abort it when it gets to MTR

**Pull request merged**